src/main_operations/modules/apriltags/pre_processing/ai_accelleration/training/augmentation/augment_data.py

```
@@ -0,0 +1,407 @@
+    """
+    # Get next frame number thread-safely
+    frame_number = get_next_frame_number()
+    base_name = f"frame_{frame_number:04d}"
+    return augmentation_func(image, annotations, base_name, output_dir)
+
+
+def process_sample_worker(
+    json_file: str,
+    augmentations_to_apply: Dict[str, Callable],
+    output_dir: str,
+    progress_bar_position: int = 0,
```

Ruff (0.14.0)

Unused function argument: `progress_bar_position`

(ARG001)